homeassistant/components/glances/sensor.py

```diff
@@ -238,6 +240,13 @@ class GlancesSensorEntityDescription(
         icon="mdi:harddisk",
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "uptime": GlancesSensorEntityDescription(
```

Comment:
On this line I think you should have a pair of values like :
("uptime", "uptime"): GlancesSensorEntityDescription(